src/tests/unit/common/element-snapshot-formatter.test.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+    CSS_MODULE_HASH_REPLACEMENT,
+    formatHtmlForSnapshot,
+    normalizeCssModuleClassNames,
+    normalizeOfficeFabricGeneratedClassNames,
+} from 'tests/common/element-snapshot-formatter';
+
+describe('element-snapshot-formatter', () => {
+    describe('normalize office fabric generated class names', () => {
+        describe.each`
+            actualClassName              | expectedClassName
+            ${'thumb-123'}               | ${'thumb-000'}
+            ${'ms-Panel-234'}            | ${'ms-Panel-000'}
+            ${'thumb-321 ms-Panel-432'}  | ${'thumb-000 ms-Panel-000'}
+            ${'no-numbers'}              | ${'no-numbers'}
+            ${'no-numbers thumb-543'}    | ${'no-numbers thumb-000'}
+            ${'Panel765-word'}           | ${'Panel000-word'}
+            ${'Panel765-word thumb-123'} | ${'Panel000-word thumb-000'}
```

Comment:
I think this is missing a test case that would fail for `thumb-123 name-not-ending-in-digit`

Reply:
Added